javascript/src/main/components/ChannelMessageReaction/MessageTableReaction.js

```diff
@@ -23,6 +23,12 @@ export default ({ messages=[],reaction }) => {
     ];
 
     return (
-        <BootstrapTable keyField='ts' data={messages} columns={columns}/>
+        
+        <BootstrapTable 
+   bootstrap4={true}
+   keyField='id' 
+   data={messages} 
+   columns={columns} 
+/>
```

Comment:
```suggestion
        <BootstrapTable 
            bootstrap4={true}
            keyField='id' 
            data={messages} 
            columns={columns} 
        />
```

Can we fix the spacing for the `BootstrapTable` here (as well as everywhere else in the PR you made this change)?

Reply:
Yes. It will be done in next two days.

Reply:
I have fixed all spacing problems.